### PR TITLE
Define `size` for SVD

### PR DESCRIPTION
**Pull request by mohdibntarek**
Closes https://github.com/JuliaLang/julia/issues/28438. And first PR!

### CODE REVIEW

**Comment by ararslan** [comment]
Welcome to Julia! Given @fredrikekre's comment in the linked issue, is this pull request still relevant? If so, it would be great if you could add a test.

**Comment by mohamed82008** [comment]
My motivation for this PR was:
```julia
julia> using LinearAlgebra

julia> ldiv!(zeros(3), svd(rand(3,3)), rand(3))
ERROR: MethodError: no method matching size(::SVD{Float64,Float64,Array{Float64,2}}, ::Int64)
Closest candidates are:
  size(::Any, ::Integer, ::Integer) at deprecated.jl:53
  size(::Any, ::Integer, ::Integer, ::Integer...) at deprecated.jl:53
  size(::BitArray{1}, ::Any) at bitarray.jl:74
  ...
Stacktrace:
 [1] ldiv!(::Array{Float64,1}, ::SVD{Float64,Float64,Array{Float64,2}}, ::Array{Float64,1}) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v0.7\LinearAlgebra\src\factorization.jl:101
 [2] top-level scope at none:0
```
The above works after defining `size` for `SVD`. So I think it's relevant yes. `size` is defined for QR and Cholesky factorizations but not SVD.

File: stdlib/LinearAlgebra/test/svd.jl [hunk 1]
```diff
@@ -29,6 +29,8 @@ using LinearAlgebra: BlasComplex, BlasFloat, BlasReal, QRPivoted
     # matrices from the factorization as expected.
     @test sf1.U*Diagonal(sf1.S)*sf1.Vt' ≊ m1
     @test sf2.U*Diagonal(sf2.S)*sf2.Vt' ≊ m2
+
```
Comment (DokFaust) [inline stdlib/LinearAlgebra/test/svd.jl]: The indentation here is making CI fail, as it is superfluous. Extra white spaces are not allowed as they make diffs harder to read.

**Comment by ararslan** [comment]
Should this be included in 0.7?

**Comment by andreasnoack** [comment]
> Should this be included in 0.7?

Yes, I've added the backport label.

**Pull request merged**